src/Pages/CardEditor.js

```diff
@@ -25,10 +27,14 @@ export const CardEditor = () => {
 		changeShapePosition,
 		changeShapeSize,
 		changeImagePosition,
-		changeImageSize
+		changeImageSize,
+		// pdfUrl,
+		setPDFUrl
 	} = useStage()
 
-	const [redirectUrl, setredirectUrl] = useState(null)
+	let navigate = useNavigate()
+
+	// const [redirectUrl, setredirectUrl] = useState(null)
```

Comment:
Let's erase this, probably move onto checkout component for manual redirects

Reply:
I erased line 37